src/universal/components/Action/Action.js

```diff
@@ -26,7 +28,13 @@ const Action = (props) => {
       <Toast />
       <SocketHealthMonitor />
       <Switch>
-        <Route exact path="/" component={LandingContainer} />
+        {__RELEASE_FLAGS__.newSignIn
```

Comment:
Minor, minor quibble; should this be called `__RELEASE_FLAGS__` (plural) or `__RELEASE_FLAG_` (singular)? I think the latter, right? It's a single boolean value, right?

Reply:
Oh crap, I'm an idiot. Nevermind. I see that this is actually an object exposed in `releaseFlags.js`!